src/commcare_cloud/ansible/group_vars/all.yml

```diff
@@ -138,6 +138,7 @@ BANK_ROUTING_NUMBER_WIRE: "{{ localsettings_private.BANK_ROUTING_NUMBER_WIRE | d
 BITLY_APIKEY: "{{ localsettings_private.BITLY_APIKEY | default('') }}"
 BITLY_LOGIN: "{{ localsettings_private.BITLY_LOGIN | default('') }}"
 BOOKKEEPER_CONTACT_EMAILS: "{{ localsettings_private.BOOKKEEPER_CONTACT_EMAILS | default(None) }}"
+CELERY_RESULT_BACKEND: 'djcelery.backends.database:DatabaseBackend'
```

Comment:
I left this in here for now, wasn't clear if there was a reason it isn't in `settings.py`